resources/views/searches/resources/people/index.blade.php

```diff
@@ -1,5 +1,21 @@
 <!-- Content - OPEN -->
 <div class="container margin-top padding-bottom">
-    <img src="/img/mockup_people_involved.png" style="width: 100%" />
+    <div class="text-right">
+        <!-- Add Involved People - OPEN -->
+        <button id="add_involved_person" data-toggle="modal" data-target="#add_modal" class="btn btn-outline-primary margin-left align-right btn-sm margin-bottom" role="button"
+           <?php if ($search->status == 1 || Auth::user()->profile == 'guest'){ ?> style="display: none" <?php } ?> >
+            <span class="octicon octicon-plus"></span>
+            {{ __('actions.add') . ' ' . __('main.people_involved') }}
+        </button>
+        <!-- Add Involved People - CLOSE -->
+        @if(Session::has('sucecss'))
+            <div class="alert alert-success">{{ Session::get('success') }}</div>
+        @endif
```

Comment:
Alerts it is already implemented. It needs to go to the top and not use any code, just use the include for this: 
`
    @include('parts.alerts')
`